tests/test_examples.py

```diff
@@ -37,8 +38,7 @@ def test_array_content(self, rst_and_truevals: tuple[Raster, np.ndarray]):
         np.random.seed(42)
         values = np.random.choice(rst.data.data.flatten(), size=5, replace=False)
 
-        # TODO: understand why the values differ slightly between different OS or python configurations
-        assert values == pytest.approx(truevals, abs=1e-3)
+        assert values == pytest.approx(truevals)
```

Comment:
what is the absolute/relative error tolerated by default?

Reply:
By default it's relative 1e-6